enhancements/update/update-status-api.md

```diff
@@ -0,0 +1,398 @@
+ip-10-0-92-180.us-east-2.compute.internal   Outdated     Pending   4.14.0-rc.3   ?
+
+= Worker Upgrade =
+
+WORKER POOL   ASSESSMENT   COMPLETION   STATUS
+worker        Pending      0% (0/3)     2 Available, 0 Progressing, 0 Draining
+
+Worker Pool Nodes: worker
+NAME                                        ASSESSMENT   PHASE     VERSION       EST   MESSAGE
+ip-10-0-20-162.us-east-2.compute.internal   Unavailable  Pending   ?             ?     Not ready
+ip-10-0-4-159.us-east-2.compute.internal    Outdated     Pending   4.14.0-rc.3   ?
+ip-10-0-99-40.us-east-2.compute.internal    Outdated     Pending   4.14.0-rc.3   ?
+
+= Update Health =
+SINCE     LEVEL     IMPACT             MESSAGE
+58m18s    Error     API Availability   Cluster Operator kube-apiserver is degraded (NodeController_MasterNodesReady)
+1h0m17s   Error     API Availability   Cluster Operator control-plane-machine-set is unavailable (UnavailableReplicas)
+now       Warning   Update Stalled     Cluster Version version is failing to proceed with the update (ClusterOperatorsDegraded)
+
+Run with --details=health for additional description and links to related online documentation
+```
+
+The example illustrates the proposal to translate the OpenShift state closer to a typical administrator's mental model. Nodes flipping their versions from old to new is something that humans expect to see in an update, but assigning a version to a `Node` is not a trivial operation in OpenShift because a version is just one of the metadata items on a configuration that MCO is reconciling on Nodes. For MCO, this is a simple config change to reconcile, not an abstract "update."
+
+All functionality will be delivered under the new `UpdateStatus` [Capability][cluster-capabilities] so that clusters that do not want the functionality do not need to spend resources on running the new controller. The Capability will be a part of the `vCurrent` and `v4.X` capability sets, which means the functionality will be enabled by default, and admins need to opt-out at installation or update time. Like all capabilities, once enabled, it cannot be disabled.
+
+This proposal intends to capture only a limited part of the roadmap for a system reporting update health and status. The [Update Health API Controller Proposal][update-health-api-dd] document describes a potential future system where individual (and possibly external) domain-responsible components would produce update insights into the `UpdateStatus` API. This enhancement proposal corresponds to only Stage 1 of the implementation roadmap outlined there. There has yet to be a consensus on further stages, and this enhancement does not aim to achieve it. Any hypothetical modular system will be a much larger effort to implement, and its design will benefit from experience and feedback obtained by seeing API consumed by users, backed by the proposed simple implementation in the USC.
+
+### Workflow Description
+
```

Comment:
any chance to capture a driving use case in this proposal illustrated by concrete insights which would not possible with todays UX?